otter/integration/tests/test_convergence.py

```diff
@@ -993,6 +993,47 @@ def test_false_negative_on_server_create_from_nova_no_overshoot(self):
                       matcher=HasLength(2), timeout=600)
         return d
 
+    @skip_me("Skipped until this error transition is fixed in Otter #1510")
+    @skip_if(not_mimic, "This requires Mimic for error injection.")
+    @tag("CATC-024")
+    @inlineCallbacks
+    def test_various_nova_40xs(self):
+        """
+        CATC-024
+
+        Validate that an autoscale group transitions to an ERROR state when
+        a scaling operation results in a 400: BadRequest response from Nova.
+
+        """
+        group, server_name_prefix = self.helper.create_group(
+            image_ref=image_ref, flavor_ref=flavor_ref,
+            min_entities=3, max_entities=10,
+            server_name_prefix="nova_400s"
+        )
+        mimic_nova = MimicNova(pool=self.helper.pool, test_case=self)
+
+        message_400 = "Can not find requested image"
+
+        yield mimic_nova.sequenced_behaviors(
+            self.rcs,
+            criteria=[{"server_name": server_name_prefix + ".*"}],
+            behaviors=[
+                {"name": "default"},
+                {"name": "fail",
+                 "parameters": {"code": 400, "message": message_400,
+                                "type": "bad_request"}}
```

Comment:
As per some discussion in slack, probably any unknown 4XX should put it into ERROR.  In which case, do we want to have 2 test cases (one for unknowns, one for knowns)?  

And which (if any) do we want to make this one?